packages/web/src/flash/FlashContext.tsx

```diff
@@ -1,64 +1,28 @@
-import React, { createContext, useReducer, useMemo } from 'react'
+import toast from 'react-hot-toast'
 
-import type { FlashMessage } from 'src/flash/FlashReducer'
-import FlashReducer from 'src/flash/FlashReducer'
-
-type Message = { id: number; text: string }
-type MessageOptions = Omit<FlashMessage, 'text' | 'id'>
-
-type FlashContext = {
-  messages: Message[]
-  addMessage(text: string, options?: MessageOptions): void
-  dismissMessage(messageId: number): void
-  cycleMessage(messageId: number): void
+/** @deprecated addMessage is deprecated and will be removed in RedwoodJS v1.0. Please update your components to use the `toast()` API: https://react-hot-toast.com/docs/toast */
+const addMessage = (text: string) => {
```

Comment:
Same as above, should we still have `options`?

Reply:
@Tobbe I don't have enough context to add these, could you help me out?

Reply:
@peterp sure thing. I'll try to get it done after the kids hava gone to bed tonight

Reply:
Ok - let me chat with @thedavidprice. We might actually just remove Flash completely!

I think I'm going to merge this now, and we can decide what to do when we release, since technically we should have a code freeze today :P